stacks_queues/queue.py

```diff
@@ -15,47 +18,82 @@ def __init__(self):
         self.front = -1
         self.rear = -1
         self.size = 0
-      
 
     def enqueue(self, element):
         """ Adds an element to the Queue
             Raises a QueueFullException if all elements
             In the store are occupied
             returns None
         """
-        pass
+        if self.size == INITIAL_QUEUE_SIZE:
```

Comment:
I would suggest comparing to `buffer_size` instead of your constant. That way, if you were ever to expand or adapt this class to have a buffer size that could increase or decrease, you code would still work without needing to change all references to `INITIAL_QUEUE_SIZE` to the new buffer size.
```suggestion
        if self.size == self.buffer_size:
```